api/src/main/java/com/viaversion/viaversion/api/protocol/version/ProtocolVersion.java

```diff
@@ -139,34 +139,38 @@ public static boolean isRegistered(final VersionType versionType, final int vers
     }
 
     public static boolean isRegistered(int version) {
-        return isRegistered(VersionType.RELEASE, version);
+        for (final VersionType versionType : VERSIONS.keySet()) {
```

Comment:
This is breaking API and there are people depending on this specific behaviour.

Reply:
This one should be transparent to the API-depending projects, since it will just expand the range of protocol versions which return _true_ for supported if (and only if) some ViaVersion extensions register them in the `VERSIONS` map

Reply:
This will still break people who depend on this method returning false for non release versions. We don't want to break API if there is no immediate reason to.